README.md

```diff
@@ -128,6 +129,25 @@ Several libraries can be included in single `%use` statement, separated by `,`:
 ```
 %use lets-plot, krangl, mysql(8.0.15)
 ```
+You can also specify the source of library descriptor. By default, it's downloaded from the latest commit on the
+branch which kernel was built from. If you want to try descriptor from another revision, use the following syntax:
+```
+// Specify tag
+%use lets-plot@0.8.2.5
+// Specify commit sha, with more verbose syntax
+%use lets-plot@ref[24a040fe22335648885b106e2f4ddd63b4d49469]
+// Specify git ref along with library arguments
+%use krangl@dev(0.10)
+```
+Other options are resolving library descriptor from a local file or from remote URL:
+```
+// Load library from file
+%use mylib@file[/home/user/lib.json]
+// Load library descriptor from a remote URL
+%use herlib@url[https://site.com/lib.json]
+// You may omit library name for file and URL resolution:
+%use @file[lib.json]
```

Comment:
In this case, the path is relative. Is it relative to the current notebook scope or to notebook parent directory? I think that the later is preferred since one can ship descriptors alongside the notebook.

Reply:
It's relative to current working directory, which is the directory from where you start your jupyter client

Reply:
It would make sense if it would be relative to the notebook file. This way it won't matter where you start the notebook.

Reply:
As I can figure out from [this SO answer](https://stackoverflow.com/a/52119628/5488904) it's impossible to obtain the notebook path without extending the messaging protocol :( Regarding your offer about resolution against parent directory, I think it's a bit unnatural: can't really figure out the case. Anyway, you may go to the parent directory via `..`, and it will work

Reply:
OK then. My idea was to be able to put descriptor in the same directory as the notebook itself and do not think about where do you start notebook from, but it is not so important to waste effort.